src/alloc/isolated_alloc.rs

```diff
@@ -52,20 +52,18 @@ impl IsolatedAlloc {
         Layout::from_size_align(size, align).unwrap()
     }
 
-    /// Returns the layout used to allocate the pages that hold small allocations.
+    /// For greater-than-page-sized allocations, returns the minimum allocation
+    /// size to make sure the alignment requested can be given.
     #[inline]
-    fn page_layout(&self) -> Layout {
-        Layout::from_size_align(self.page_size, self.page_size).unwrap()
-    }
-
-    /// If the allocation is greater than a page, then round to the nearest page #.
-    #[inline]
-    fn huge_normalized_layout(layout: Layout, page_size: usize) -> Layout {
+    fn huge_normalized_layout(&self, layout: Layout, page_size: usize) -> usize {
```

Comment:
Since this takes `self`, why does it need `page_size` separately?

Reply:
oops